src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -480,6 +490,21 @@ private String getDoi(String doi) {
         return doi;
     }
 
+    private String getArxivId(String arxivId) {
+        int pos;
+        if (arxivId == null) {
+            pos = curString.indexOf("arxiv");
+            if (pos < 0) {
+                pos = curString.indexOf("arXiv");
+            }
+            if (pos >= 0) {
+                String arxivText = curString.substring(pos);
+                return ArXivIdentifier.parse(arxivText).map(ArXivIdentifier::asString).orElse(null);
+            }
+        }
+        return arxivId;
+    }
```

Comment:
No! We have the class org.jabref.model.entry.identifier.ArXivIdentifier use this.

Reply:
This is still unfixed.